examples/react/getting-started/src/components/Posts.tsx

```diff
@@ -0,0 +1,71 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+import styles from 'scss/components/Posts.module.scss';
+import { Post } from 'types';
+import Heading, { HeadingProps } from './Heading';
+import { PostListLoader } from 'components';
+
+interface Props {
+  posts: Post[] | undefined;
+  intro?: string;
+  id?: string;
+  heading?: string;
+  headingLevel?: HeadingProps['level'];
+  postTitleLevel?: HeadingProps['level'];
+  readMoreText?: string;
+  isLoading: boolean;
+}
+
+function Posts({
+  posts,
+  intro,
+  heading,
+  id,
+  headingLevel = 'h1',
+  postTitleLevel = 'h2',
+  readMoreText = 'Read more',
+  isLoading,
+}: Props): JSX.Element {
+  return (
+    // eslint-disable-next-line react/jsx-props-no-spreading
+    <section className={styles['posts-block']} {...(id && { id })}>
+      <div className="wrap">
+        {heading && (
+          <Heading level={headingLevel} className={styles.heading}>
+            {heading}
+          </Heading>
+        )}
+        {intro && <p className={styles.intro}>{intro}</p>}
+        <div className="posts">
+          {isLoading && <PostListLoader />}
+          {posts &&
+            posts.map((post) => (
+              <div
+                className={styles.single}
+                key={post.id}
```

Comment:
Should be `key={post.id ?? ''}` to avoid issues while post is being fetched and `id` is undefined.